### PR TITLE
netty: fix ENHANCE_YOUR_CALM debug data logging

### PR DESCRIPTION
**Pull request by akandybaev**

### CODE REVIEW

File: netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java [hunk 1]
```diff
@@ -459,6 +466,37 @@ public void run() {
...
+      public void close() throws SecurityException {
+      }
+    };
+    Logger logger = Logger.getLogger(NettyClientHandler.class.getName());
+    try {
+      logger.addHandler(handler);
+      enqueue(newCreateStreamCommand(grpcHeaders, streamTransportState));
+      channelRead(goAwayFrame(0, 11 /* Enhance your calm */,
+            Unpooled.copiedBuffer("this is a test", UTF_8)));
+      assertNotNull(logRef);
```
Comment (creamsoup) [inline netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java]: thanks for the fix! it looks like this should test `logRef.get()`?

File: netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java [hunk 2]
```diff
@@ -459,6 +465,37 @@ public void run() {
...
+      @Override
+      public void close() throws SecurityException {
+      }
+    };
+    Logger logger = Logger.getLogger(NettyClientHandler.class.getName());
+    try {
+      logger.addHandler(handler);
+      enqueue(newCreateStreamCommand(grpcHeaders, streamTransportState));
+      channelRead(goAwayFrame(0, 11 /* Enhance your calm */,
+            Unpooled.copiedBuffer("this is a test", UTF_8)));
```
Comment (creamsoup) [inline netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java]:
```suggestion
          Unpooled.copiedBuffer("this is a test", UTF_8)));
```

File: netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java [hunk 3]
```diff
@@ -459,6 +465,37 @@ public void run() {
...
+      assertNotNull(logRef.get());
+      assertTrue(MessageFormat.format(logRef.get().getMessage(), logRef.get().getParameters())
+              .contains("Debug data: this is a test"));
```
Comment (creamsoup) [inline netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java]:
```suggestion
          .contains("Debug data: this is a test"));
```

File: netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java [hunk 4]
```diff
@@ -84,10 +84,16 @@
...
+import java.text.MessageFormat;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Queue;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Handler;
+import java.util.logging.LogRecord;
+import java.util.logging.Logger;
+
```
Comment (voidzcy) [inline netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java]: Can you delete this empty line?

**Comment by voidzcy** [comment]
Thanks for contributing to grpc-java.

**Pull request merged**